### PR TITLE
Fixing Issue #74108/ Adding the Bomb Testing Site to the anomaly placer blacklist

### LINKED ISSUE #74108: Pyro anomalies can spawn in the bomb testing region
## Round ID:
[202091](https://scrubby.melonmesa.com/round/202091)

## Testmerges:
any

## Reproduction:
pyro anomaly slimes can spawn in the bomb testing region, which they cannot escape.

### PR DESCRIPTION
**Pull request by Moth-lantern**
## About The Pull Request
Fixes #74108  it seemed easy enough to fix so I thought I'd knock it out really quick
## Why It's Good For The Game
Was kind of frustrating that an anomaly could appear in a place where it wouldn't affect anything, now that's gone!
## Changelog
:cl:
fix: Anomalies will no longer spawn at the bomb testing range satellite.
/:cl:

### CODE REVIEW

File: code/modules/events/anomaly/anomaly_placer.dm [hunk 1]
```diff
@@ -65,6 +65,7 @@
 		/area/station/solars,
 		/area/station/holodeck,
 		/area/station/maintenance,
+		/area/station/science/ordnance/bomb,
```
Comment (san7890) [inline code/modules/events/anomaly/anomaly_placer.dm:68]: can you alphabetize this list? there's a few extensions in vscode that do this for you. thanks

Comment (Moth-lantern) [reply code/modules/events/anomaly/anomaly_placer.dm:68]: yep sorry will do!

Comment (san7890) [reply code/modules/events/anomaly/anomaly_placer.dm]: no worries this sort of thing isn't really written down anywhere it's just a nicety

**Review by san7890 — APPROVED** [review]
fyi: i updated your changelog since it's meant to be player facing, and your explanation was way too technical+confusing for the average player. the new one is much more usable to anyone with a basic knowledge of both anomalies and the bomb testing range satellite (a much greater chunk of the playerbase)

**Pull request merged**